Updates db troubleshooting (#1591)
mulesoft/docs-connectors: db/1.10/modules/ROOT/pages/database-connector-troubleshooting.adoc

	</flow>
----

== Understand Select and Stored Procedure Operations Warning Exceptions

> When the Database Connector *Select* and *Stored procedure* operations execute correctly but results are not present, Mule runtime engine (Mule) returns a `WARN` exception message instead of throwing the exception.

Reviewer: ```suggestion
When the Database Connector *Select* and *Stored procedure* operations execute correctly but do not return results, Mule runtime engine (Mule) returns a `WARN` exception message instead of throwing the exception.
```

> This occurs when executing an invalid SQL statement or expecting that an error within a set of statements will collapse the procedure. In fact, JDBC returns a series of `ResultSets`, `UpdateCounts`, and `Exceptions`.

Reviewer: I'm sure this is right, but I don't know what "expecting that an error within a set of statements will collapse the procedure" means.

> For the *Select* operation, the behavior occurs when executing an invalid SQL statement, and for the *Store procedure* operation when there is an error within a set of statements that can collapse the procedure. In fact, JDBC returns a series of `ResultSets`, `UpdateCounts`, and `Exceptions`.

Reviewer: ```suggestion
For the *Select* operation, the behavior occurs when executing an invalid SQL statement, and for the *Store procedure* operation,  when there is an error within a set of statements that can collapse the procedure. In fact, JDBC returns a series of `ResultSets`, `UpdateCounts`, and `Exceptions`.
```
